distribution/changelog.txt

```diff
@@ -2,6 +2,8 @@
 ------------------------------------------------------------------------
 - Feature: [#13000] objective_options command for console
 - Fix: [#3200] Close Construction window upon selecting vehicle page
+- Improved: [#13023] Made add_news_item console command last argument, assoc, optional.
```

Comment:
The changelog is ordered by issue/PR number, grouped by category in the following order:

- Feature
- Change
- Fix
- Improved
- Technical